### ISSUE TITLE
sourceMap URL request return != 200

### ISSUE THREAD

**Issue by molitona** [denandz/sourcemapper#10]
even if sourcemap exist here but it showed non 200 status, why ?
```
sourcemapper -output srcc -url https://www.carrefour.fr/js/users-rating.7a441ab84b1aaa5e3943.js.map
[+] Retrieving Sourcemap from https://www.carrefour.fr/js/users-rating.7a441ab84b1aaa5e3943.js.map.
sourceMap URL request return != 200
```

**Comment by denandz** [comment]
This is a cool edge case. That URL returns a 403 status code, but still includes the response body! 

I think I can get sourcemapper to check for a body length, alert the user about the non-200 error code, then proceed.

**Comment by denandz** [comment]
Apologies, helps if I actually look at the response properly. The web server in this case is serving you a generic 403 error page, instead of a sourcemap:

```
HTTP/2 403 
date: Mon, 24 Jul 2023 03:39:29 GMT
content-type: text/html; charset=UTF-8
cache-control: max-age=15
expires: Mon, 24 Jul 2023 03:39:44 GMT
set-cookie: __cf_bm=2hp5T6V9oPU6zp7AnLIl8JITGFsPt5tAUTYzDmssZFw-1690169969-0-Adprpme4ejhzuYY57pJ8vLakAt8veoftCc57u9gvtohgiCg67QWGIRHcOmSPj7UMnhr9ACdtgLK92MycBinvqpM=; path=/; expires=Mon, 24-Jul-23 04:09:29 GMT; domain=.carrefour.fr; HttpOnly; Secure; SameSite=None
strict-transport-security: max-age=2592000
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
referrer-policy: strict-origin-when-cross-origin
server: cloudflare
cf-ray: 7eb922e3efe9eea6-AKL

<!DOCTYPE html>
<html lang="fr">
<head>
<title>Carrefour</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
```

Still though, edge cases where non 200-responses still erroneously include the legit response are not unheard of, so will tweak the error logic

**Comment by denandz** [comment]
closed by 0adc84ca6e050f8f3b3fbb616c490f77e79b0528

**Comment by molitona** [comment]
nice fix @denandz 

https://alumni-friends.monash.edu/ascendportal/s/sfsites/auraFW/javascript/YlFCb0tteDV4aGhQNENwMlB3WlU0ZzE0VDFFNVRwUjllNDRPNDJ3WDdmcWcyNDQuMjAuNC0yLjQxLjQ/aura_prod.js.map
returns 
invalid character ')' looking for beginning of value
why ?
